tests/Integration/Behaviour/Features/Context/Domain/TaxRulesGroupFeatureContext.php

```diff
@@ -79,4 +92,190 @@ public function assertLastErrorIsTaxRulesGroupNotFound(): void
     {
         $this->assertLastErrorIs(TaxRulesGroupNotFoundException::class);
     }
+
+    /**
+     * @When I add new tax rules group :taxRulesGroupReference with following properties:
+     */
+    public function createTaxRulesGroup(string $taxRulesGroupReference, TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+
+        $command = new AddTaxRulesGroupCommand(
+            $data['name'],
+            PrimitiveUtils::castStringBooleanIntoBoolean($data['is_enabled'])
+        );
+
+        /** @var TaxRulesGroupId $taxRulesGroupId */
+        $taxRulesGroupId = $this->getCommandBus()->handle($command);
+
+        SharedStorage::getStorage()->set($taxRulesGroupReference, $taxRulesGroupId->getValue());
+    }
+
+    /**
+     * @When I edit tax rules group :taxRulesGroupReference with following properties:
+     */
+    public function editTaxRulesGroup(string $taxRulesGroupReference, TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+
+        $taxRulesGroupId = SharedStorage::getStorage()->get($taxRulesGroupReference);
+        $command = new EditTaxRulesGroupCommand($taxRulesGroupId);
+        if (isset($data['name'])) {
+            $command->setName($data['name']);
+        }
+        if (isset($data['is_enabled'])) {
+            $command->setEnabled(PrimitiveUtils::castStringBooleanIntoBoolean($data['is_enabled']));
+        }
+        $this->getCommandBus()->handle($command);
+    }
+
+    /**
+     * @When /^I (enable|disable)? tax rules group "(.*)"$/
+     */
+    public function toggleStatusTaxRulesGroup(string $action, string $taxRulesGroupReference): void
+    {
+        $expectedStatus = 'enable' === $action;
```

Comment:
can we put these `enable`, `enabled`, `disabled` status in consts so that it's not hardcoded at several places?

(is it normal that we have both `enable` and `disabled` ?)

Reply:
actually it's not necessary, those are feature context behat functions, wording comes from behat scenarios 👍